classify & colorize all components and pages

phillipet opened the pull request.

phillipet committed:
remove style css of btn-reg

phillipet committed:
change all style of pages/comps to stylus language

change 601px to $breakpoint-xs quasar, qcardWidth of Login & Register form 350px to 390px

phillipet committed:
update CHANGELOG

phillipet committed:
Globalize class css 

move common class each pages & components to app.styl

phillipet committed:
remove css un-use var

phillipet committed:
format and apply newline at the end of stylus files

phillipet committed:
colorizle css login and register page

phillipet committed:
Merge branch 'develop' into feature/change-style-to-stylus

phillipet committed:
Merge remote-tracking branch 'upstream/develop' into feature/change-style-to-stylus

phillipet committed:
colorize reg and login page, add color facebook varriable

phillipet committed:
classify css style for category comp

phillipet committed:
colorize menu, menu detail page, menu comp,  add color text-modifier variable

phillipet committed:
colorize counter comp, add bg-facebook class again (class name is generated by generator of color attribute quasar tag)

phillipet committed:
classify css for news, news-detail comp

colorize news. news-detail comp, change color reg, login page, remove some class, color variable...

phillipet committed:
format code

hemvanh commented on src/components/News.vue:
```diff
@@ -4,7 +4,7 @@
         <div class="subtitle text-white q-ml-sm q-mt-sm">{{item.subTitle}}</div>
         <div class="q-display-1 text-white q-ml-sm q-mb-sm">{{item.title}}</div>
-        <q-btn class=" fixed-bottom-left q-ml-sm q-mb-sm" @click="show" :label="item.label" :class="{ hidden: item.isBtnHidden }" color="white" text-color="brown" rounded>
+        <q-btn class="fixed-bottom-left q-ml-sm q-mb-sm" @click="show" :label="item.label" :class="{ hidden: item.isBtnHidden }" color="secondary" text-color="white" rounded>
```
color="secondary" text-color="white"   -> <q-btn class="fixed-bottom-left q-ml-sm q-mb-sm bg-secondary text-neutral" ?

phillipet replied:
only text-primary, text-secondary, text-tertiary are available, text-neutral can not apply

hemvanh commented on src/components/NewsDetail.vue:
```diff
@@ -29,5 +29,22 @@ export default {
+  color $primary
+
+.news-detail-content-color
+  color grey
```
color grey  -> can change to use color $variable ? like: color $neutral

phong-et replied:
I only use setting color variable when it is used least twice times in app. 
Example : facebook color (it is used ast Register and Login page)

phillipet replied:
Ok, I will use $neutral=grey

hemvanh replied:
$neutral is just an example, currently $neutral is #E0E1E2
the main point is to try our best to NOT introduce new colors, use primary/secondary/.... instead
unless it's really necessary to

hemvanh commented on src/pages/MenuDetail.vue:
```diff
@@ -105,5 +105,7 @@ export default {
-<style>
+<style scoped lang="stylus">
+.text-modifier
+  color #616161
```
color #616161 possible to use color $variable ?

phillipet replied:
I only use setting color variable when it is used least twice times in app.
Example : facebook color (it is used ast Register and Login page)

hemvanh replied:
yes it should be like that with other css styling properies, but with color, we try to stick around with quasar pre-defined name like primary,secondary,.... so that we could control the app color theme at one place, for #616161, can we just fall back to use primary/secondary... ? unless #616161 is a very important color for the page.

phillipet committed:
re-colorize page and component

- remove text-modifier, replace by secondary
- set color $neutral to grey code color (use for popup news-detail text content at home page)
